DigitalLearningSolutions.Data.Tests/Services/TrackerActionServiceTests.cs

```diff
@@ -181,11 +181,17 @@ public void StoreDiagnosticJson_returns_success_response_if_successful()
 
         [Test]

```

Comment:
`[Test]` is not strictly required when you also have `[TestCase(...)]` defined.

Reply:
ah cool, removed